distribution/changelog.txt

```diff
@@ -8,7 +9,7 @@
 - Feature: Add ride console command for diagnostics and changing vehicle type.
 - Feature: Allow selecting corners when using the mountain tool.
 - Feature: Allow setting ownership of map edges.
-- Feature: Allow up to 255 cars per train.
+- Feature: Allow up to 255 cars per train and 32 track units per station
```

Comment:
that was just something i noticed wasn't added into the changelog back when it was implemented. I should probably make that a seperate pull request.